tools/postgres/schema.sql

```diff
@@ -42,8 +41,8 @@ CREATE TABLE webknossos.annotations(
   _task CHAR(24),
   _team CHAR(24) NOT NULL,
   _user CHAR(24) NOT NULL,
-  tracing_id CHAR(36) NOT NULL UNIQUE,
-  tracing_typ webknossos.ANNOTATION_TRACING_TYPE NOT NULL,
+  skeletonTracingId CHAR(36) UNIQUE,
+  volumeTracingId CHAR(36) UNIQUE,
```

Comment:
@fm3 Please add a comment here, that this _must_ be unique over the two columns, and is currently assured by the datastore, which generates those ids.